main/tests/cypress/cypress/e2e/project/grid/column/reconcile/actions/Add column_with_URLs_of_matched_entities.cy.js

```diff
@@ -0,0 +1,25 @@
+describe('Add Column With Url Of Matched Entities', () => {
+    afterEach(() => {
+    cy.addProjectForDeletion();
+    });
+    it('Add Column With Url Of Matched Entities', () => {
+    cy.visitOpenRefine();
+    cy.navigateTo('Import project');
+    cy.get('.grid-layout').should('to.contain', 'Locate an existing Refine project file');
+    
+    //we're using here the reconciled and mateched project, so rows are reconciled and matched to their best candidate
+    cy.get('#project-tar-file-input').attachFile('openrefine-demo-sandbox-csv.openrefine.tar.gz.zip')
+    cy.get('#import-project-button').click();
+    cy.getCell(0, 'Item').should('to.contain', 'Choose new match');
+    cy.columnActionClick('Item', [
+    'Reconcile',
+    'Actions',
+    'Add column with URLs of matched entities',
+    ]);
+    cy.assertNotificationContainingText(
+        'Create new column Url of Matched entities based on column Item by filling 4 rows with if(cell.recon.match!=null,"https://www.wikidata.org/wiki/{{id}}".replace("{{id}}",escape(cell.recon.match.id,"url")),"")',
+    );
+    cy.getCell(0, 'Url of Matched entities').should('to.contain', 'https://www.wikidata.org/wiki/Q3938');
```

Comment:
I think it would make sense to add other assertions in the current test for the empty cells in other rows: that does not cost us much.
To test the scenario of service not having a view URL, it seems more complicated as we are currently not isolating reconciliation services (our tests rely on the live endpoints). So we would need to have a public endpoint which can be trusted not to have a view template in the future, or re-introduce a local service that is run alongside the tests (which @fgiroud moved out of because the tests were not passing reliably if I remember correctly).
Overall, given the high cost of Cypress tests, I'd be inclined not to add too much complexity there: we should not use them as unit tests. In the future it'd be nice to have some sort of a system for frontend unit tests, if we were to migrate to a framework which makes that possible.